bigtable/client/table.cc

```diff
@@ -45,9 +45,9 @@ namespace {
 
 namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
-// Call the `google.bigtable.v2.Bigtable.MutateRow` RPC repeatedly until
-// successful, or until the policies in effect tell us to stop.
-void Table::Apply(SingleRowMutation&& mut) {
+namespace noex {
+std::vector<FailedMutation> Table::Apply(SingleRowMutation&& mut,
```

Comment:
There are fairly detailed comments about the implementation of `Apply()` and `BulkApply()` please move those comments here, since the implementation has moved.